website/content/docs/concepts/jobs.mdx

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+page_title: Jobs
+description: |-
+  Learn about Nomad's Job feature, which is how you deploy your apps.
+---
+
+# Jobs
+
+This page explains the concept of a job, which is the Nomad construct for
+deploying your app and its configuration.
```

Comment:
I don't know if others feel the same, but I've tended to shy away from the term "app" or application for what Nomad jobs do. We've used terms like workload or task in the past, but they also have their drawbacks. The implication of running an App is certainly suited to what service and system jobs _could_ be, but they also could be things like maintenance scripts, etc. In other words: I'm apprehensive about this word because it seems to sell things a little short!

Reply:
Ok, I understand your point and will clarify on this page. Application/app is currently used throughout the docs and tutorial, so that's a cleanup for another day.